### PR TITLE
Update reference docs to sdk@v0.15.7, core@master

### PR DESCRIPTION
**Pull request by davidwallacejackson**
Used https://github.com/wandb/wandb/commit/dd20d99f06efbe4bcf6abda05cf436740d843059

### CODE REVIEW

File: docs/ref/python/log.md [hunk 1]
```diff
@@ -46,12 +40,11 @@ for all of the different supported types or check out our
 [guides to logging](https://docs.wandb.ai/guides/track/log) for examples,
 from 3D molecular structures and segmentation masks to PR curves and histograms.
 `wandb.Table`s can be used to logged structured data. See our
-[guide to logging tables](https://docs.wandb.ai/guides/tables/log-tables)
```
Comment (dmitryduev) [inline docs/ref/python/log.md:49]: this link is broken

File: docs/ref/python/log.md [hunk 2]
```diff
@@ -46,12 +40,11 @@ for all of the different supported types or check out our
...
+[guide to logging tables](https://docs.wandb.ai/guides/data-vis/log-tables)
```
Comment (dmitryduev) [inline docs/ref/python/log.md:43]: this works, but redirects to https://docs.wandb.ai/guides/track/log/log-tables

Comment (ngrayluna) [reply docs/ref/python/log.md:43]: That's correct. I am about to merge that renaming (data-vis -> tables)

Comment (dmitryduev) [reply docs/ref/python/log.md:43]: now that the automation works, it should be fixed in the SDK docstrings.

File: docs/ref/python/log.md (hunk 1 above)
Comment (ngrayluna) [reply docs/ref/python/log.md:49]: Renaming data-vis -> tables . Redirects will take care of this. :)

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**